### PR TITLE
Add flag to control if chartmuseum returns absolute or relative URLs

### PR DESCRIPTION
**Pull request by darend**
## what

Add flag to control if chartmuseum returns absolute or relative URLs

This relates to the following PRs:

- #150 and #165 - Previous attempt to enable this behavior
- https://github.com/goharbor/harbor/pull/7719
- https://github.com/goharbor/harbor/issues/6572

### CODE REVIEW

File: templates/chartmuseum/chartmuseum-cm.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ data:
   CACHE_REDIS_ADDR: "{{ template "harbor.redis.host" . }}:{{ template "harbor.redis.port" . }}"
   CACHE_REDIS_DB: "{{ template "harbor.redis.chartmuseumDatabaseIndex" . }}"
   BASIC_AUTH_USER: "chart_controller"
+{{- if eq .Values.chartmuseum.absoluteUrl }}
```
Comment (ywk253100) [inline templates/chartmuseum/chartmuseum-cm.yaml]: I think the `eq` should be removed as it needs two parameters.

**Comment by ywk253100** [comment]
@

File: templates/chartmuseum/chartmuseum-cm.yaml (hunk 1 above)
Comment (darend) [reply templates/chartmuseum/chartmuseum-cm.yaml]: Oops!

Comment (darend) [reply templates/chartmuseum/chartmuseum-cm.yaml]: @ywk253100 fixed in f7bcd80b34c5623555efbd2ec82e6cbe8430129b

**Comment by ywk253100** [comment]
@darend Thanks

**Pull request merged**